airbyte-integrations/connectors/destination-s3/src/test/resources/parquet/json_schema_converter/json_conversion_test_cases.json

```diff
@@ -742,23 +742,18 @@
           "name": "date_time_field",
           "type": [
             "null",
-            { "type": "long", "logicalType": "timestamp-micros" },
-            "string"
+            { "type": "long", "logicalType": "timestamp-micros" }
           ],
           "default": null
         },
         {
           "name": "date_field",
-          "type": ["null", { "type": "int", "logicalType": "date" }, "string"],
+          "type": ["null", { "type": "int", "logicalType": "date" }],
           "default": null
         },
         {
           "name": "time_field",
-          "type": [
-            "null",
-            { "type": "long", "logicalType": "time-micros" },
-            "string"
-          ],
+          "type": ["null", { "type": "long", "logicalType": "time-micros" }],
```

Comment:
We need the `string` here as the default type because sometimes the data is not that clean, and the Json built-in format cannot be properly parsed. So those invalid values will remain strings. This ensures that we can sync as much data as possible.

The solution is to duplicate the Json to Avro schema converter, and create one just for BigQuery. In that way, none of the test cases for pure Avro will be in your way, and the changes can be reverted for this file.